examples/conversion_reaction/mainConversionReaction.m

```diff
@@ -166,39 +160,58 @@
 % optionsMultistartMeigo.localOptimizer = 'meigo-ess';
 % optionsMultistartMeigo.localOptimizerOptions = MeigoOptions;
 % optionsMultistartMeigo.n_starts = 2;
-% parametersMeigo = getMultiStarts(parameters, objectiveFunction, optionsMultistartMeigo);
+% parameters = getMultiStarts(parameters, objectiveFunction, optionsMultistartMeigo);
 
 % This section uses PSwarm, a particle swarm optimizer
 % (Install from http://www.norg.uminho.pt/aivaz/pswarm/ and uncomment)
-%
+
 % optionsMultistartPSwarm = optionsMultistart.copy();
 % optionsMultistartPSwarm.localOptimizer = 'pswarm';
 % optionsMultistartPSwarm.n_starts = 10;
-% parametersPSwarm = getMultiStarts(parameters, objectiveFunction, optionsMultistartPSwarm);
+% parameters = getMultiStarts(parameters, objectiveFunction, optionsMultistartPSwarm);
+
+
+%% Visualization of fit
+% The measured data is visualized in plot, together with fit for the best
+% parameter value found during getMultiStarts
+
+if strcmp(optionsMultistart.mode,'visual')
+    % Simulation
+    tsim = linspace(t(1),t(end),100);
+    ysim = simulateConversionReaction(exp(parameters.MS.par(:,1)),tsim);
+
+    % Plot: Fit
+    figure('Name','Conversion reaction: Visualization of fit');
+    plot(t,y,'bo'); hold on;
+    plot(tsim,ysim,'r-'); 
+    xlabel('time t');
+    ylabel('output y');
+    legend('data','fit');
+end
 
 
 %% Profile likelihood calculation -- Parameters
 % The uncertainty of the estimated parameters is visualized by computing
 % and plotting profile likelihoods. In getParameterProfiles, this is done
-% by using repeated reoptimization
+% by using repeated reoptimization, if standard setings are used.
 parameters = getParameterProfiles(parameters, objectiveFunction, optionsMultistart);
 
 
 %% Markov Chain Monte Carlo sampling -- Parameters
 % Values for the parameters are sampled by using an Parallel Tempering (PT)
 % algorithm. This way, the underlying probability density of the parameter 
-% distribution can be captured. 
+% distribution can be captured. Since only one temperature is used, this is
+% effectively an adapted Metropolis algorithm single-chain algorithm.
 
 % Building a struct covering all sampling options:
 optionsSampling = PestoSamplingOptions();
-optionsSampling.rndSeed     = 3;
-optionsSampling.nIterations = 2e3;
+optionsSampling.nIterations = 1e4;
+optionsSampling.mode = 'visual';
 
 % PT specific options:
 optionsSampling.samplingAlgorithm   = 'PT';
-optionsSampling.PT.nTemps           = 3;
-optionsSampling.PT.exponentT        = 4;    
```

Comment:
It's supposed to be only one temperature, there's hence no temperature exponent needed...